debian/libxapianVERSION-dev.install

```diff
@@ -1,7 +1,6 @@
 usr/bin/xapian-config-1.3
 usr/include
 usr/lib/cmake/xapian
-usr/lib/libxapian-1.3.a
```

Comment:
I'm a bit at a loss as to what this file even does. Maybe it can be removed?

@wgreenberg You and Jasper were the last ones to touch the debian packaging, do you remember?

Reply:
The file is also part of the upstream debian packaging, so I think this is OK.